Core/include/Acts/Propagator/ConstrainedStep.hpp

```diff
@@ -56,12 +54,8 @@ class ConstrainedStep {
   constexpr ConstrainedStep() = default;
 
   /// constructor from Scalar
-  /// navigation direction is inferred by the sign of the step size
   /// @param value is the user given initial value
-  constexpr explicit ConstrainedStep(Scalar value) {
-    m_values[user] = std::abs(value);
-    m_direction = Direction::fromScalar(value);
-  }
+  constexpr explicit ConstrainedStep(Scalar value) { m_values[user] = value; }
```

Comment:
Since step size values are not supposed to always be positive, it might make sense to have assertions around checking that whenever something sets a step size value.

Reply:
my though was that you might intentionally create an object that is in back stepping mode. not sure if that is used or useful tho

Reply:
Wait, I thought back stepping mode was now supposed to be handled via state.stepping.navDir?

Reply:
no sorry for the confusion - I would keep the back stepping sign but remove. I would only remove the propagation direction sign because the `ConstrainedStep` is invariant for forward or backward